doc/users/faq/howto_faq.rst

```diff
@@ -9,6 +9,61 @@ How-to
 .. contents::
    :backlinks: none
 
+
+.. _how-to-too-many-ticks:
+
+Why do I have so many ticks, and/or why are they out of order?
+--------------------------------------------------------------
+
+One common cause for unexpected tick behavior is passing a list of strings
+instead of numbers or datetime objects. This can easily happen without notice
```

Comment:
Optional: Should we markup the reason? This could help when quickly skimming the explantion.

```suggestion
One common cause for unexpected tick behavior is passing a *list of strings
instead of numbers or datetime objects*. This can easily happen without notice
```
Alternatively even bold?